etna/lib/etna/clients/magma/models.rb

```diff
@@ -438,7 +445,7 @@ def self.copy(source, dest)
           dest.format_hint = source.format_hint
           dest.hidden = source.hidden
           dest.link_model_name = source.link_model_name
-          dest.read_only = source.read_only
```

Comment:
Is there a reason to remove the read_only flag while copying an attribute?

Reply:
Nice catch! Just a foolish error...